templates/components/scope.html

```diff
@@ -6,9 +6,32 @@
 Please see LICENSE files in the repository root for full details.
 -#}
 
+{# Macro to remove 'unsafe' scope from a scope list. Usage:
```

Comment:
...I think it's actually removing the safe ones but preserving the unsafe?

Reply:
Fixed in e26a7f3c862ff1307e5ee75d3997b355c136a74b